.github/workflows/erlang.yml

```
@@ -0,0 +1,25 @@
+name: Erlang CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    container:
+      image: erlang:22.0.7
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Compile
+      run: rebar3 compile
+    - name: Format check
+      run: rebar3 format --verify
+    - name: Run tests
+      run: rebar3 do eunit, ct
```

We have a `test` target for this.
```suggestion
      run: rebar3 test
```

---

yes but it outputs coverage, np?

---

no problem… if GA eventually understands how to interpret and check that… that would be even better! To ensure that we don't reduce coverage, I mean.